Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -207,9 +207,9 @@ private void InitializeNotifyIcon()
 
         private void InitProgressbarAnimation()
         {
-            var da = new DoubleAnimation(ProgressBar.X2, ActualWidth + 100,
```

Comment:
It's because the animation has to go outside window. Anyway, it was fixed to add +100 to the actual width.

Reply:
yeah and i dont understand why add 100 lol, magic number

Reply:
>  I don't think I'll make a theme with a larger width, so I just put in a fixed value and it came out well, so I did it. The color was also more noticeably adjusted.

if you are reverting back to using ActualWidth, will it erase this improvement you mentioned about the color and noticeability of the progress animation?

Reply:
da1's 100 number is progressbar width size. (it is not gauge.) it will move left to right actual window size. 
one more  add 100 (200) is to make it go outside altogether. (Before I fixed it, it was stopped befre go outside.)  I don't know if I understood because I didn't make it, but it works as my intended now.

Reply:
Sry, No. it's not moving properly. I'll look a bit more.